lib/android-helpers.js

```diff
@@ -373,13 +373,23 @@ helpers.unlock = async function (adb) {
 
   await retryInterval(10, 1000, async () => {
     logger.debug("Screen is locked, trying to unlock");
-    await adb.startApp({
+
+    // first manually stop the unlock activity
+    await adb.forceStop('io.appium.unlock');
+
+    // then start the app twice, as once is flakey
+    let startOpts = {
       pkg: "io.appium.unlock",
       activity: ".Unlock",
       action: "android.intent.action.MAIN",
       category: "android.intent.category.LAUNCHER",
-      flags: "0x10200000"
-    });
+      flags: "0x10200000",
+      stopApp: false
+    };
+    await adb.startApp(startOpts);
+    await adb.startApp(startOpts);
+
+    // wait a moment, and check if it worked
     await sleep(1000);
```

Comment:
seeing this code---i know it's not part of this PR but don't you think we should use a retryInterval rather than a static sleep here?